src/compiler/handler.go

```diff
@@ -66,7 +67,7 @@ func (l *Language) Handler(c *websocket.Conn) {
 		return
 	}
 
-	timeoutContext, cancel := context.WithTimeout(context.Background(), 0)
+	timeoutContext, _ := context.WithTimeout(context.Background(), 0)
```

Comment:
The 0 should have been changed, since this will always terminate instantly.

Reply:
Not Cancel will cause memory leak and panic.

Reply:
Could have used deferred call to cancel here.
```
defer cancel()
```

Reply:
Yes it will cause memory leak. We can use deferred call to cancel